.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "plugin-ceno/pyFreenet"]
 	path = plugin-ceno/pyFreenet
-	url = git@github.com:misaakidis/pyFreenet.git
+	url = https://github.com/misaakidis/pyFreenet.git
```

Comment:
Travis CI can't clone over git:// by default, probably an env var defaulting to ssh://.